File: public/phase-2-20160620022203/JAVASCRIPT/snowflakes.js
// Name : Snowflakes
// input:
// n : length of array
// k : number of  letters of the English alphabet
// english_letters :  contains n uppercase English letters
var min_number_of_changes,changed_array;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var k = parseInt(elements[1], 10);
var english_letters = readline();


// write your code here
// store your results in `min_number_of_changes`,`changed_array`

// Dummy Data
min_number_of_changes = 2;
changed_array = "ABCACA";


// output
print(min_number_of_changes);
print(changed_array);

File: public/phase-2-20160620022203/JAVA/snowflakes.java
// Name : Snowflakes
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : length of array
		// k : number of  letters of the English alphabet
		// english_letters :  contains n uppercase English letters
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int k = Integer.parseInt(elements[1]);
		String english_letters = in.nextLine();


		// write your code here
		// store your results in `min_number_of_changes`,`changed_array`

		// output
		// Dummy Data
		int min_number_of_changes=2;
		String changed_array="ABCACA";

		System.out.println(min_number_of_changes);
		System.out.println(changed_array);

	}
}